---
layout: single
title: Research
classes: wide
description:
permalink: /research/
---

The Slotkin laboratory investigates how plant cells determine which fragments of DNA should be expressed (like genes), and which should be targeted for repression and not allowed to express. The regions of the genome that are targeted for repression are called Transposable Elements (TEs), which are mobile ‘jumping genes’ that cause mutations when they jump. Most of the time these new mutations generated by TEs hurt the cell or are inconsequential, but occasionally something new is produced that is helpful and will be selected for, leading to some spectacular examples of TE-induced gene or trait regulation.  

In a broad sense, what the Slotkin laboratory aims to determine is how self-perpetuating feed-forward cycles in the cell are initiated. The cycles we work on are RNA interference (RNAi), RNA-directed DNA Methylation (RdDM) and maintenance epigenetic silencing. We have learned that these cycles can feed into each other. Our goal now is to determine how the first cycle is triggered to initiate.

![image-center]({{ site.url }}{{ site.baseurl }}/assets/images/Three_cycles.png){: .align-center}

Projects in the lab vary from proteomics to genomics, bench-work to purely computational, and basic fundamental questions to applied crop pest resistance. The majority of our research is done on the powerful model plant Arabidopsis, while through collaborations we have also concentrated on maize, soybeans and aphids, and bioinformatically on any plant genome. The unifying features of all of our projects are based on our in-lab expertise on: Transposable Elements, Plant Epigenetics, RNAi, RdDM, small RNAs, DNA methylation and the Bioinformatics of multi-copy regions of the genome.

If you need help with one of these topics, please feel free to email us and request a collaboration!

See our [Publications](/publications/), [Press](/press/) and [Resources](/tools-resources) pages for links to our research articles, while these six below are good representations of our opinions and viewpoints:

Shahid, S., Slotkin, R. (2020). [The current revolution in transposable element biology enabled by long reads.](https://dx.doi.org/10.1016/j.pbi.2019.12.012) Current Opinion in Plant Biology 54, 49-56.

Slotkin, R. (2018). [The case for not masking away repetitive DNA.](https://dx.doi.org/10.1186/s13100-018-0120-9) Mobile DNA 9, 475.

Cuerda-Gil, D., Slotkin, R. (2016). [Non-canonical RNA-directed DNA methylation.](https://dx.doi.org/10.1038/nplants.2016.163) Nature Plants 2(11), 16163.

Sigman, M., Slotkin, R. (2016). [The First Rule of Plant Transposable Element Silencing: Location, Location, Location.](https://dx.doi.org/10.1105/tpc.15.00869) The Plant Cell 28(2), 304-313.

Fultz, D., Choudury, S., Slotkin, R. (2015). [Silencing of Active Transposable Elements in Plants.](https://dx.doi.org/10.1016/j.pbi.2015.05.027) Current Opinion in Plant Biology 27, 67-76.

Martinez, G., Slotkin, R. (2012). [Developmental relaxation of transposable element silencing in plants: functional or byproduct?](https://dx.doi.org/10.1016/j.pbi.2012.09.001Geneticists) Current Opinion in Plant Biology 15, 496-502.
