---
layout: single
classes: wide
title: Tools & Resources
description:
permalink: /tools-resources/
---
Below are some of the tools and resources developed by the Slotkin Lab:

#### Image Resources for plant growth under high CO2 conditions

From: (2023) K. Panda, B. Mohanasundaram, J. Gutierrez, L. McLain, S.E. Castillo, H. Sheng, A. Casto, G. Gratacós, A. Chakrabarti, N. Fahlgren,  S. Pandey,  M.A. Gehan and R.K. Slotkin. [The plant response to high CO2 levels is heritable and orchestrated by DNA methylation](https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.18876). New Phytologist
Images available [here](https://github.com/danforthcenter/heritable-highCO2-response).

#### Resources for *in vivo* Protein-RNA tethering 

Plasmids and seeds available from the ABRC: [Link](https://abrc.osu.edu/stocks?search%5Btaxon%5D=Arabidopsis+thaliana&search%5Bsearch_text%5D=Slotkin&search%5Bsearch_fields%5D=Donor+last+name)
From: [A plant tethering system for the functional study of protein-RNA interactions in vivo](https://plantmethods.biomedcentral.com/articles/10.1186/s13007-022-00907-w) (2022). D. Cuerda-Gil, Y.-H. Hung, K. Panda and R.K. Slotkin. Plant Methods 18: 75.

#### Improved annotation of Arabidopsis Transposable Elements

[Long-read cDNA Sequencing Enables a ‘Gene-Like’ Transcript Annotation of Arabidopsis Transposable Elements.](http://www.plantcell.org/content/32/9/2687) K. Panda and R.K. Slotkin.  *Annotations are available on [GitHub](https://github.com/KaushikPanda1/AthalianaTETranscripts).*

---
#### Software Tools

Tool to determine the stength of RNA-directed DNA Methylation (RdDM). From: [An siRNA-guided Argonaute protein directs RNA Polymerase V for the first round of RNA-directed DNA methylation](https://www.nature.com/articles/s41477-021-01008-7) (2021). M. Sigman, K. Panda, R. Kirchner, L.L. McLain, H. Payne, J.R. Peasari, A.Y. Husbands, R.K. Slotkin, A.D. McCue.Nature Plants 7: 1461-1474. 
*Custom python script to analyze dot-plot images from KisMeth available from [GitHub](https://github.com/jpeasari/Dot-Plot-Anaysis-OpenCV).

[EpiTEome: Simultaneous detection of transposable element insertion sites and their DNA methylation levels.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1232-0) J. Daron and R.K. Slotkin. Genome Biology v18:7704.  *Available on [GitHub](https://github.com/jdaron/epiTEome).*

![image-right]({{ site.url }}{{ site.baseurl }}/assets/images/kismeth-1.jpg){: .align-right}
[Kismeth: Analyzer of Plant Methylation States Through Bisulfite Sequencing.](http://www.biomedcentral.com/1471-2105/9/371) E. Gruntman<sup>\*</sup>, Y. Qi<sup>\*</sup>, R.K. Slotkin<sup>\*</sup>, T. Roeder, R.A. Martienssen and R. Sachidanandam. BMC Bioinformatics v9: e371.<sup> \*</sup>*These authors contributed equally to this manuscript. Available as an [web-based tool.](http://katahdin.mssm.edu/kismeth/revpage.pl)*

---
#### Teaching Resources

R.K. Slotkin. [Designing a Better Laboratory Course.](http://gsi.berkeley.edu/slotkinrk-2005/) **Teaching Guide for Graduate Student Instructors** 2005-2006, University of California Press.

---
#### Outreach

[Mutant Millets outreach](https://mutantmillets.org), serving St. Louis region High Schools.
<figure>
  <a href="https://mutantmillets.org"><img src="https://mutantmillets.files.wordpress.com/2014/01/cropped-header4.jpg"></a>
</figure>
